lib/helpers.bash

```diff
@@ -224,6 +224,9 @@ _bash-it-update-() {
 
   cd "${BASH_IT}" || return
 
+  DIFF=$(git diff --name-status)
+  [ ${#DIFF} -gt 0 ] && echo -e "Either commit your changes using git commit, or clean them...\n$DIFF" && return 1
+
```

Comment:
Yeah I think `-n` and `-z` are standard for length checks:

> -n string  | True if the length of string is non-zero.

so I'd go with `[ -n "${DIFF}" ]` for consistency

Reply:
@davidpfarrell Done